package.json

```diff
@@ -22,18 +22,18 @@
         "cheerio": "~1.0.0-rc.2",
         "chromedriver": "~2.37.0",
         "eslint": "~4.19.1",
-        "eslint-config-prettier": "~2.9.0",
+        "eslint-config-prettier": "^2.9.0",
         "eslint-plugin-import": "~2.10.0",
-        "eslint-plugin-prettier": "~2.6.0",
+        "eslint-plugin-prettier": "^2.6.0",
         "eslint-plugin-react": "~7.7.0",
         "express": "~4.16.3",
         "full-icu": "~1.2.1",
         "jest": "20.0.4",
         "lerna": "~2.9.1",
         "lolex": "~2.3.2",
         "mocha": "~5.0.5",
-        "prettier": "~1.7.0",
-        "prettier-eslint-cli": "~4.7.1",
+        "prettier": "^1.11.1",
```

Comment:
we prefer `~` over `^`, please update the file.

Also, you need to commit the `yarn.lock` changes.